Update content.md (#1090)
arduino/docs-content: content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md

You can now freely program your Portenta X8 Linux OS. In the sections below you can check some basic commands to get started.

> ### Change default user password

Reviewer: ```suggestion
### Change Default User Password
```

You can now freely program your Portenta X8 Linux OS. In the sections below you can check some basic commands to get started.

### Change default user password

> Your X8 comes with the default user fio with password fio.

Reviewer: ```suggestion
Your Portenta X8 comes with the default user fio with password fio.
```

> For security reasons, we strongly suggest to change the default password. To do so, when logged in into your X8, launch this command to change the 

Reviewer: ```suggestion
For security reasons, we strongly suggest changing the default password. To do so, when logged in to your Portenta X8, launch this command to change the password of the fio account:
```

> password of the fio account:

Reviewer: ```suggestion
```
